package.json

```diff
@@ -27,12 +27,13 @@
   "homepage": "https://github.com/Cimpress-MCP/lambda-essentials-ts#readme",
   "dependencies": {
     "aws-sdk": "2.939.0",
-    "axios": "0.21.1",
+    "axios": "0.21.2",
```

Comment:
Upgraded to fix https://github.com/Cimpress-MCP/lambda-essentials-ts/security/dependabot security vulnerabilities.